bluemira/codes/_freecadapi.py

```diff
@@ -1997,7 +1997,7 @@ def collect_verts_faces(
         # tesselation is likely to be the most expensive part of this
         v, f = face.tessellate(tesselation)
 
-        if v != []:
+        if v != [] and f != []:
             verts.append(np.array(v))
             faces.append(np.array(f) + voffset)
             voffset += len(v)
```

Comment:
Another option, but I'm not sure we want faces that have vertices but not faces..
```suggestion
        if v != []:
            verts.append(np.array(v))
            if f != []:
                faces.append(np.array(f) + voffset)
            voffset += len(v)
```

Reply:
this could be more robust though

Reply:
I don't think I know enough about how this works to comment, really. Maybe wires will have vertices but not faces? This looks sensible to me though